server/database/config/db_build.sql

```diff
@@ -0,0 +1,67 @@
+BEGIN;
+  DROP TABLE IF EXISTS employees,questions,employees_questions,forms,employee_form,form_questions
+  CASCADE;
+
+
+
+CREATE TABLE employees
+(
+  id SERIAL PRIMARY KEY,
+  name VARCHAR(50) NOT NULL,
+  email VARCHAR  ,
+  image TEXT,
+  date date,
+  position VARCHAR(50),
+  role VARCHAR
+);
+
+CREATE TABLE questions
+(
+  id SERIAL PRIMARY KEY,
+  employee_id INT,
+  FOREIGN KEY (employee_id)  REFERENCES employees(id),
+  type VARCHAR NOT NULL,
+  result_rate VARCHAR ,
+  context TEXT
```

Comment:
another thing, we see to add an answer column in the question table, true?

Reply:
this column has been added.